site/en/docs/privacy-sandbox/attribution-reporting/getting-started/index.md

```diff
@@ -0,0 +1,227 @@
+
+Summary reports are not tied to a specific event on the ad side. These reports provide richer, higher-fidelity conversion data than event-level reports. A combination of privacy techniques help reduce the risk of identity joining across sites.
+
+#### Aggregatable reports - additional concepts
+
+In addition to understanding source, triggers, and headers as as noted for event-level reports, for aggregatable reports you'll need to review some further concepts.
+
+While you may not need to delve into all of the details, such as exactly how [noise](/docs/privacy-sandbox/attribution-reporting/understanding-noise) is generated, a solid understanding of the basics will help you develop a plan. Knowledge of these concepts is not strictly necessary for implementation, but to understand and refine the reports generated, you'll benefit from this information.
+
+These concepts are:
+
+- [Contribution budget](/docs/privacy-sandbox/attribution-reporting/contribution-budget)
+- [Noise](/docs/privacy-sandbox/attribution-reporting/understanding-noise)
+
+
+## Implementation
+
+You can get started by trying out the steps that follow, and then adapt your strategy to fit, or you can plan your strategy in advance.
+
+{% Aside 'important' %}
+Before you can register sources and triggers and get reports, your sites need to have SSL/TLS certificates; in other words, you need to be running over HTTPS.
+{% endAside %}
+
+### Event-level report generation
+
+Once you're familiar with the concepts for [event-level reports](/docs/privacy-sandbox/attribution-reporting/report-types) and have defined your [sources](#source), [triggers](#trigger), set your headers, and defined your [endpoints](#endpoints), you're ready to receive event-level reports.
+
+1. **Register a source.** Refer to [Registering a source](/docs/privacy-sandbox/attribution-reporting/register-attribution-source).
+   
+   In the `Attribution-Reporting-Register-Source` header, add the necessary fields to generate event-level reports. These fields include:
+
+   - `source_event_id`
+   - `destination`
+   - `expiry` (optional)
+   - `debug_key`
+   - `debug_reporting`
+   - `priority`
+
+1. **Register a trigger** refer to [Triggering Attribution](https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#triggering-attribution).
+    
+   In your `Attribution-Reporting-Register-Trigger`  header, include
+
+   - `event_trigger_data` Note that if you omit this, event-level reports will not be generated.
+    
+   Here's an example header:
+
+    ```javascript
+    {
+      "event_trigger_data": [{
+      "trigger_data": "[unsigned 64-bit integer]",
+      "priority": "[signed 64-bit integer]",
+      "deduplication_key": "[unsigned 64-bit integer]"
```

Comment:
from the header validator: ["deduplication_key"] unknown field. (https://wicg.github.io/attribution-reporting-api/validate-headers)

Reply:
And no more errors...